local-cli/server/runServer.js

```diff
@@ -157,7 +157,7 @@ function getPackagerServer(args, config) {
     maxWorkers: args.maxWorkers,
     platforms: defaultPlatforms.concat(args.platforms),
     polyfillModuleNames: config.getPolyfillModuleNames(),
-    polyfills: config.polyfills,
+    getPolyfills: config.getPolyfills,
```

Comment:
@mjesun The reason I would like to not call it here is so that Metro can call it from `Resolver#_getPolyfillDependencies` and pass the platform (ios or android) into `config.getPolyfills`. (In case it wasn't clear, getPolyfills is `({platform: string}) => Array<string>`.)